src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let price = days * 40;
+
+  if (days >= 3 && days <= 6) {
+    price = price - 20;
+  } else if (days >= 7) {
+    price = price - 50;
```

Comment:
Replace the literals '3', '6', '7', '20', and '50' with named constants to make the code more readable and maintainable.